datasette/views/table.py

```diff
@@ -581,18 +593,23 @@ async def data(
         _next = _next or special_args.get("_next")
         offset = ""
         if _next:
+            next_params_not_parsed = _next.split(",")
             if is_view:
                 # _next is an offset
                 offset = f" offset {int(_next)}"
             else:
                 components = urlsafe_components(_next)
                 # If a sort order is applied, the first of these is the sort value
-                if sort or sort_desc:
-                    sort_value = components[0]
-                    # Special case for if non-urlencoded first token was $null
-                    if _next.split(",")[0] == "$null":
-                        sort_value = None
-                    components = components[1:]
+                if sort:
+                    sort_values = dict()
```

Comment:
If the order of elements matter here, you need to check what's the oldest Python version supported by Datasette - order in dicts is retained since 3.6